### PR TITLE
Species-specific Sprites

### PR DESCRIPTION
**Pull request by ghost**
Species-specific Sprite system for clothing ported from Paradise. Sprites from Bay/vg/Paradise
lizsuitss
lizsuitse
# Only contains the welding helmet lizard sprite, removed hardsuit ones

No changelog because it's not that significant.

### CODE REVIEW

File: code/modules/mob/living/carbon/human/update_icons.dm [hunk 1]
```diff
@@ -311,7 +343,14 @@ There are several things that need to be remembered:
 		if(!t_state)
 			t_state = belt.icon_state
 
-		var/image/standing = belt.build_worn_icon(state = t_state, default_layer = BELT_LAYER, default_icon_file = 'icons/mob/belt.dmi')
+		var/image/standing
+		if(belt.icon_override)
```
Comment (RemieRichards) [inline code/modules/mob/living/carbon/human/update_icons.dm]: stuff like this should all go inside of ```build_worn_icon()```
Remember to give highest priority to ```alternate_worn_icon```, as it's the admin/highest override.

**Comment by KorPhaeron** [comment]
If goof posts in this PR I'm closing guncrafting

**Comment by TehZombehz** [comment]
Gosh dang it I wanted to do this.
At least this also adds the system support.
Inb4 anti-lizard memes spew about as per the usual. (Maybe it'll even get downvoted for that aspect alone). 
@KorPhaeron  why not just bet 20 dosh with me that it happens?

**Comment by KorPhaeron** [comment]
REEEE FURRIES aside though this means it's more or less impossible for a human to disguise as a lizard or vice versa

**Comment by Supermichael777** [comment]
i hate those sprites

**Comment by Incoming5643** [comment]
No, this is bad. This is an unsustainable situation where spriters will be expected to do more and more sprites over time to mach what's already there, ala fat sprites.

Also for things that REALLY needed this I already added a way of doing this in #20599 with the clothing var mutantrace_variation. We don't need to port anything.

**Comment by Grazyn** [comment]
As it was said in OOC, they're HARDsuit. Why would they have a different shape if there's a lizard instead of a human inside? Lorewise, lizards tuck their tail inside the suit and that's it. The helmet is large enough to accomodate all their frills and gills and stuff. It's also one of the few ways for a lizard to masquerade as a human.

**Comment by PKPenguin321** [comment]
Sprites look like shit and this has the same maintainability issues fat sprites do

**Comment by uraniummeltdown** [comment]
If people don't like the sprites (either aesthetics or the disguising thing) that's fine and I could remove them, I think the system could be useful in future though

**Comment by Hatfish** [comment]
That looks horrible, though
What's wrong with the sprites we have right now?

**Comment by Hatfish** [comment]
The system itself sounds fine but those sprites are horrible.

**Comment by Kromgar** [comment]
Jesus christ their hideous

**Comment by ChangelingRain** [comment]
these are really bad sprites these are basically fat lizards and not actually lizard-shaped

**Comment by Iamgoofball** [comment]
i disapprove of this not because ree furries but because now we have to have spriters make double of things for le furry special snowflake meme

**Comment by TechnoAlchemisto** [comment]
Looks bad and is unnecessary.

**Comment by dannno** [comment]
look at those fatasses 
fucking go on a diet holy shit

**Comment by optimumtact** [comment]
Yeah I don't see the point, plus it looks weird when you put on a suit and it is a different shape depending on who put it on.

**Comment by Aloraydrel** [comment]
ugly

**Comment by PKPenguin321** [comment]
Not to mention sec hardsuit looks nothing like the regular sprite right down to color. I think the biggest atrocity though is that nuke ops hardsuit. Jesus Christ. These sprites are bad enough that I thought this was a joke PR at first

**Comment by Haevacht** [comment]
As I mentioned in OOC they are the HARDsuits, which would require specific re-contruction to have a tail section. And with lizards lorewise being shit on at every turn, NT really isn't going to let then have anything, even the engisuits are too valuable for them. They just gotta smoosh their heads and tails in.

Then again even the softsuits wouldn't be soft and pliable enough to accomodate a tail.

Why does the CMO hardsuit have a red plus. It's supposed to be cyan.

**Comment by uraniummeltdown** [comment]
Alright I'll remove the hardsuit ones but should I keep the welding helmet?

**Comment by WJohn** [comment]
The idea is cool and I support the system, but these sprites are really, really bad.

Why are they so damn fat?

I also don't know if I want hardsuit restrictions because all it does is add scarcity to already extremely rare equipment that I'd like to be made much less rare.

**Comment by optimumtact** [comment]
Remove hardsuits

**Comment by bear1ake** [comment]
Respect for overriding hardcoded icons!

**Comment by Grazyn** [comment]
Still don't like the idea of welding helmets or anything else changing shape depending on the race like in a cheap fantasy RPG

**Comment by Thunder12345** [comment]
🤢

**Comment by Cheridan** [comment]
the downthumbs win

**Pull request closed without merging**